docker/prod/generate_compose.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import os
+import jinja2
+
+# Jinja template directory
+TEMPLATE_DIR = os.path.dirname(os.path.abspath(__file__))
+TEMPLATE_FILE = "docker-compose.yml.j2"
+
+# The list of commands (passed as input)
+feeds = sys.argv[1:]
+
+if "*" in feeds:
+    print("Wildcard detected. Generating for all available feeds.")
```

Comment:
maybe should use a logger instead of print statements?

Reply:
It's actually a local script. So gonna use print to stdout